scripts/install/usr/libexec/cortx-motr/motr-server

```diff
@@ -35,6 +35,55 @@ ha_config="/etc/sysconfig/m0d-${1#m0d-}"
 kernel_config=/etc/sysconfig/motr-kernel
 service_funcs=/usr/libexec/cortx-motr/motr-service.functions
 
+# BE log resize
+NODE_NAME="/var/lib/hare/node-name"
+MD_DIR="/var/motr"
+M0BETOOL="m0betool"
+BE_REPLAY="be_recovery_run"
+BE_RESIZE="be_log_resize"
+
+service_fid=""
+
+be_log_resize() {
+   local new_be_log_size=""
+   local current_be_log_size=""
+
+   # Extract current BE log file size
+   new_be_log_size=$(grep -w MOTR_M0D_BELOG_SIZE $user_config | grep -v '#' | cut -d"=" -f2)
+   if [[ -z $new_be_log_size ]]; then
+      m0_log "BE log size is not configured"
+      return 0
+   fi
+   m0_log "Given BE log size: $new_be_log_size"
+
+   # Validate current BE LOG file size and New BE LOG file size
+   current_be_log_size=$(ls -l "$MD_DIR"/"$service_fid"/db-log/o/0:28 | awk '{ print $5 }')
+   if [[ $current_be_log_size == "" ]]; then
+      m0_log "Failed to get existing be log size"
+      return 1
+   fi
+
+   if [[ "$current_be_log_size" == "$new_be_log_size" ]]; then
+      m0_log "Current and New BE log sizes are equal"
+      return 0
+   fi
+   m0_log "Current BE log size:  $current_be_log_size"
+
+   # Replay exist BE log
+   if ! $($M0BETOOL $BE_REPLAY $MD_DIR/"$service_fid"/db); then
+      m0_log "m0d service $service_fid  log replay is failed in local node"
```

Comment:
m0_log "m0d service $service_fid  log replay is failed"

No need to mention local node.

Reply:
Done